src/test/java/de/eposcat/master/PageTest.java

```diff
@@ -0,0 +1,206 @@
+package de.eposcat.master;
+
+
+import de.eposcat.master.approachImpl.EAV_DatabaseAdapter;
+import de.eposcat.master.approachImpl.IDatabaseAdapter;
+import de.eposcat.master.connection.H2ConnectionManager;
+import de.eposcat.master.connection.RelationalApproach;
+import de.eposcat.master.exceptions.BlException;
+import de.eposcat.master.model.Attribute;
+import de.eposcat.master.model.AttributeBuilder;
+import de.eposcat.master.model.AttributeType;
+import de.eposcat.master.model.Page;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+
+import java.sql.SQLException;
+import java.util.HashMap;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.is;
+import static org.hamcrest.Matchers.not;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.fail;
+
+
+public class PageTest {
+
+    static IDatabaseAdapter dbAdapter;
+    static Attribute defaultAttribute;
+
+    @BeforeAll
+    static void initDataBase() {
+        H2ConnectionManager connectionManager = new H2ConnectionManager(RelationalApproach.EAV);
+
+        dbAdapter = new EAV_DatabaseAdapter(connectionManager);
+        defaultAttribute = new AttributeBuilder().setType(AttributeType.String).setValue("A test value").createAttribute();
+    }
+
+    @Test
+    public void createPage() {
+        try {
+            //returned Pages must have an id, the given type and no attributes
+            Page page = dbAdapter.createPage("testType");
+            assertThat(page.getId(), not(-1));
+            assertThat(page.getTypeName(), is("testType"));
+            assertThat(page.getAttributes().isEmpty(), is(true));
+
+            //fails on  missing/empty type
+            assertThrows(IllegalArgumentException.class, () -> dbAdapter.createPage(null));
+            assertThrows(IllegalArgumentException.class, () -> dbAdapter.createPage(""));
```

Comment:
These 2 checks can be added in a separate @Test method, e.g. testCreatePageWithInvalidType